tests/test_transformation/test_logcp_transformer.py

```diff
@@ -0,0 +1,112 @@
+import pandas as pd
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.transformation import LogCpTransformer
+
+_test_automatic_find_variables = [
+    (
+        "e",
+        [
+            3.6635616461296463,
+            3.6888794541139363,
+            3.6375861597263857,
+            3.6109179126442243,
+        ],
+        [
+            0.9162907318741551,
+            0.8754687373539001,
+            0.8329091229351039,
+            0.7884573603642703,
+        ],
+    ),
+    (
+        "10",
+        [1.591064607026499, 1.6020599913279625, 1.5797835966168101, 1.568201724066995],
+        [
+            0.3979400086720376,
+            0.3802112417116061,
+            0.36172783601759284,
+            0.3424226808222063,
+        ],
+    ),
+]
+
+
+@pytest.mark.parametrize("log_base, exp_age, exp_marks", _test_automatic_find_variables)
+def test_logcp_base_plus_automatically_find_variables(
+    log_base, exp_age, exp_marks, df_vartypes
+):
+    transformer = LogCpTransformer(base=log_base, variables=None)
+    X = transformer.fit_transform(df_vartypes)
+
+    # expected output
+    transf_df = df_vartypes.copy()
```

Comment:
The test file is a masterpiece. Thank you so much. 

Could we add tests for init param C and fit attribute C_?

either as part of tests 1 or 2 or separately if you find it better. Thank you!

Reply:
got it